packages/block-library/src/media-text/index.js

```diff
@@ -2,13 +2,13 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { mediaAndText as icon } from '@wordpress/icons';
```

Comment:
I wonder if it's it makes mroe sense to use `mediaAndText` as the name of the icon you import, rather than simply `icon`? It's entirely possible I'm missing some React nuance here, but what if you imported two icons?

Reply:
yes, if you import two icons it's better to keep the icon name. The advantage of using `icon` here is that you can use shorthand syntax in the block settings `icon,` instead of `icon: mediaAndText`. In the end, it's not that important though.